js/components/popupWindow.js

```diff
@@ -69,35 +72,55 @@ class PopupWindow extends ImmutableComponent {
   }
 
   render () {
-    let styles = {}
+    let style = {}
     if (parseInt(this.width)) {
-      styles.width = (parseInt(this.width) + 2)
+      style.width = (parseInt(this.width) + 2)
     }
     if (parseInt(this.height)) {
-      styles.height = (parseInt(this.height) + 2)
+      style.height = (parseInt(this.height) + 2)
     }
     if (parseInt(this.top)) {
       if (this.top + this.height < window.innerHeight) {
-        styles.top = this.top
+        style.top = this.top
       } else {
-        styles.bottom = 0
+        style.bottom = 0
       }
     }
     if (parseInt(this.left)) {
       if (this.left + this.width < window.innerWidth) {
-        styles.left = this.left
+        style.left = this.left
       } else {
-        styles.right = 0
+        style.right = '1em'
       }
     }
 
     return <div
       className={cx({
         popupWindow: true,
-        reverseExpand: styles.right !== undefined
+        [css(styles.popupWindow)]: true,
+        [css((style.right !== undefined) && styles.reverseExpand)]: true
       })}
-      style={styles} />
+      style={style} />
   }
 }
 
+const styles = StyleSheet.create({
+  popupWindow: {
+    border: `solid 1px ${globalStyles.color.gray}`,
+    boxShadow: globalStyles.shadow.flyoutDialogBoxShadow,
+    boxSizing: 'border-box',
+    color: 'black',
+    cursor: 'default',
+    display: 'flex',
+    fontSize: '11px',
+    padding: 0,
+    position: 'absolute',
+    zIndex: globalStyles.zindex.zindexPopupWindow,
+    WebkitUserSelect: 'none'
```

Comment:
`user-select` (userSelect) is supported unprefixed in Blink. We should avoid using the prefixed version.